_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -2,22 +2,23 @@
 layout: page
 title: Add a Log Drain to your App
 nav: Log Drain
-modified_at: 2019-11-18 00:00:00
+modified_at: 2020-05-22 00:00:00
 tags: Papertrail Logentries Logmatic Graylog logs integration
 ---
 
-{% include upon_request.md %}
-
 ## Integration Purposes
 
-When an application starts having a higher traffic, reading logs can be cumbersome. Using a log
-management solution to gather the logs from all your applications and all your containers eases your
-work when trying to find an information amongst all these lines of logs. A few SaaS solutions exist
-such as Logentries, Logmatic.io, Papertrail or Graylog. You can also host such solution on Scalingo.
-Here is a tutorial on [how to deploy the ELK
-stack](https://scalingo.com/articles/2018/02/23/running-the-elk-stack-on-scalingo.html) on Scalingo.
-We also support sending your applications logs using the standard syslog protocol over UDP, TCP or
-TCP+TLS.
+Using a log management solution to gather the logs from applications and
+containers eases the work when trying to find an information.
```

Comment:
We shouldn't talk about containers. We are not selling containers. We are hosting applications and databases.